packages/macos/generate_wazuh_packages.sh

```diff
@@ -168,6 +168,12 @@ function build_package() {
     # create package
     if packagesbuild ${AGENT_PKG_FILE} --build-folder ${DESTINATION} ; then
         echo "The wazuh agent package for macOS has been successfully built."
+        symbols_pkg_name=$(echo "${pkg_name}" | tr '\n' ' ' | cut -d ' ' -f 1)
+        symbols_pkg_name="${symbols_pkg_name}_debug_symbols"
+        echo "Parte deseada: ${symbols_pkg_name}"
```

Comment:
Please, translate this text.

Reply:
Done in commit: [Delete unnecessary message](https://github.com/wazuh/wazuh/pull/22642/commits/fb96c406737c373c1d43ed29963ed590dc2f4464)